support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx

```diff
@@ -120,7 +118,7 @@ const getMainDisplayPrice = (
 
 const weeklyProductProps = (
 	countryId: IsoCountry,
-	billingPeriod: WeeklyBillingPeriod,
+	billingPeriod: BillingPeriod,
```

Comment:
Are we happy to lose the tighter typing here? I think this means the TypeScript compiler wouldn't complain if you passed a billing period which didn't apply to GW (like `BillingPeriod.OneTime`)

Reply:
We can replace`WeeklyBillingPeriod` (old) with `RecurringBillingPeriod` (new) as they are the same (ie excluding `OneTime`). Updated to this.